### ISSUE TITLE
[Rule Assistant] UI lines don't meet

### ISSUE THREAD

**Issue by rmlockwood** [rmlockwood/FLExTrans#754]
In the wire diagram many of the lines don't meet or are sometimes offset. It would be nice if they joined smoothly.

![image](https://github.com/user-attachments/assets/12d6f5f3-6c6a-434b-9849-61331b604ef0)

**Comment by AndyBlack** [comment]
Yes, I agree that this is desirable.  I'm not sure I have what it takes to modify the treeflex code to make this happen, though.  I did try once and only made it worse...

**Comment by rmlockwood** [comment]
@mr-martian Is this a challenge you would be interested in?

**Comment by rmlockwood** [comment]
fixed with the new Python version of the Rule Assistant. Increasing the font everywhere by one pixel seemed to do it.
